WooCommerce/Classes/Yosemite/DefaultStoresManager.swift

```diff
@@ -351,14 +351,17 @@ private extension DefaultStoresManager {
         }
         dispatch(productSettingsAction)
 
-        group.enter()
-        let sitePlanAction = AccountAction.synchronizeSitePlan(siteID: siteID) { result in
-            if case let .failure(error) = result {
-                errors.append(error)
+        /// skips synchronizing site plan if logged in with WPOrg credentials
```

Comment:
nit: might be worth mentioning why? is it because the site plan is for WPCOM users only?

Reply:
Yes, the site plan API is a Dotcom endpoint and we cannot handle this requires for WPOrg-authenticated users. I updated the comment in 419896cc7945d582cd75d33f6aeb489d368664fb.